pkg/machines/components/diskEdit.jsx

```diff
@@ -208,6 +203,7 @@ class EditDiskModalBody extends React.Component {
                     <Modal.Title> {'Edit ' + getDiskPrettyName(vm.disks[disk.target]) + ' Attributes'} </Modal.Title>
                 </Modal.Header>
                 <Modal.Body>
+                    { showFooterWarning() }
```

Comment:
'showFooterWarning` that does not show warning in footer but in body :) That is of course just detail ;)